src/renderer.rs

```diff
@@ -667,8 +679,8 @@ pub fn render_error(
                     <script>
                         // read theme from local storage and apply it to body
                         var theme = localStorage.getItem('theme');
-                        if (theme != null && theme != 'default') {
-                            document.body.classList.add('theme_' + theme);
+                        if (theme && theme != "default") {
+                            body.setAttribute("data-theme", name)
                         }
```

Comment:
Isn't this what `updateColorScheme()` does already?

Reply:
Yup. I just figured there was probably a good reason the script wasn't in the header so both the content and error pages could use it. I didn't realize these changes broke the error page theming so this needed fixing anyways